### PR TITLE
Add more greeting

### PR DESCRIPTION
**Pull request by maxnoe**
This adds a lot of more greetings to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,10 @@
 
 Hello Michael
 
+Hello Lukas
+
+Hello Dom
```
Comment (dneise) [inline README.md]: I think an exclamation mark is needed here

**Review by dneise — CHANGES_REQUESTED** [review]

**Comment by relleums** [comment]
I like it a lot.

**Comment by relleums** [comment]
Komm klar Neise! Du brauchst kein exclamation mark!

**Review by dneise — APPROVED** [review]

**Comment by maxnoe** [comment]
Ach lass ihn doch

**Pull request merged**